### DISCUSSION TITLE
mania "fade in" mod makes holds darker and hard to read

### DISCUSSION THREAD

**Discussion by ZeroAurora** [ppy/osu#22482] [Q&A] [answered]
## Detail
When "fade in" mode is selected, the color of hold notes becomes significantly darker, making them harder to recognize. Is it intended or a mistake?
Running 2023.123.0, bg dim is 80%
## Screenshots
No mod selected:
![osu_2023-02-01_21-54-05](https://user-images.githubusercontent.com/25359455/216062106-9be10f76-6d01-43cc-9f2e-40292468da30.jpg)
![osu_2023-02-01_21-44-35](https://user-images.githubusercontent.com/25359455/216061008-fcbf1973-0b9c-43e4-a8dd-df83f02517f7.jpg)
Fade-in selected (coverage 0.3):
![osu_2023-02-01_21-55-04](https://user-images.githubusercontent.com/25359455/216062337-5c70f381-31b7-446f-b911-fd935a9c9e81.jpg)
![osu_2023-02-01_21-56-09](https://user-images.githubusercontent.com/25359455/216062596-950a08ce-b4cc-48ec-8ad7-0a5729064ead.jpg)

**Comment by ZeroAurora** [comment]
In my opinion, an ideal solution is to sharpen the color conversion process, from dark to bright.
I'm not an native English speaker and I'm sleepy right now and can't type more words `sorry :)`, so I make some drawing on figure 3 above:
![draw](https://user-images.githubusercontent.com/25359455/216068056-a5a325f1-d2ee-4019-befd-3e9d1d68d5d2.jpg)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/22485
